config/crd/patches/webhook_in_webhooktests.yaml

```diff
@@ -16,3 +16,4 @@ spec:
         namespace: webhook-operator-system
         name: webhook-service
         path: /convert
+        port: 443
```

Comment:
Without this, I get a message like this:

```
WARN[0000] Warning: Value webhook.operators.coreos.io/v2, Kind=WebhookTest: provided API should have an example annotation 
ERRO[0000] Error: Field spec.conversion.webhookClientConfig.service.port, Value 0: spec.conversion.webhookClientConfig.service.port: Invalid value: 0: port is not valid: must be between 1 and 65535, inclusive 
make: *** [Makefile:115: bundle] Error 1
```

Reply:
I'll update the CRD to use a pointer to an int and default to 443.

Reply:
This is a bug that I'll look into - we should be defaulting to 443.